src/desktop/components/react/stitch_components/ReactionArtwork.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
```

Comment:
I went ahead and refactored all of the stuff in global_react_modules into their own TS files in this directory

Reply:
very nice.